### PR TITLE
Spike/download config from s3

### PR DESCRIPTION
**Pull request by feba-rajan**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update
- [x] Spike

## Description

This PR is for the timeboxed spike to enable configuration deployment across different environments.

## Related Tickets & Documents

- Ticket Link: https://github.com/orgs/digital-land/projects/9/views/5?pane=issue&itemId=84357146&issue=digital-land%7Cconfig%7C295

## Added/updated tests?
_We encourage you to keep the code coverage percentage at 80% and above._

- [ ] Yes
- [x] No, and this is why: setting up the configuration deployment process, no new functionality added.
- [ ] I need help with writing tests

## [optional] Are there any dependencies on other PRs or Work?
PR:https://github.com/digital-land/config/pull/294

### CODE REVIEW

File: makerules.mk [hunk 1]
```diff
@@ -16,7 +16,7 @@ MAKERULES_URL=$(SOURCE_URL)makerules/main/
 endif
 
 ifeq ($(CONFIG_URL),)
-CONFIG_URL=https://raw.githubusercontent.com/digital-land/config/main/
+CONFIG_URL=https://files.development.digital-land.info/config/
```
[the remark this answers is not in the document]
Comment (cjohns-scottlogic) [reply makerules.mk:19]: isn't the environment set in the makerules somewhere?  $(ENVIRONMENT) maybe?

File: pipeline.mk [hunk 2]
```diff
@@ -232,14 +232,12 @@ datasette:	metadata.json
 	--load-extension $(SPATIALITE_EXTENSION) \
 	--metadata metadata.json
 
-FALLBACK_CONFIG_URL := https://files.planning.data.gov.uk/config/pipeline/$(COLLECTION_NAME)/
-
 $(PIPELINE_DIR)%.csv:
 	@mkdir -p $(PIPELINE_DIR)
 	@if [ ! -f $@ ]; then \
-		echo "Config file $@ not found locally. Attempting to download..."; \
+		echo "Config file $@ not found locally. Attempting to download from s3...."; \
```
[the remark this answers is not in the document]
Comment (cjohns-scottlogic) [reply pipeline.mk:238]: I think we still want to be able to run from a checkout of the config repo. @eveleighoj ?

File: makerules.mk (hunk 1 above)
[the remark this answers is not in the document]
Comment (eveleighoj) [reply makerules.mk:19]: Yep this should be changed to Production for merge

File: pipeline.mk (hunk 2 above)
[the remark this answers is not in the document]
Comment (eveleighoj) [reply pipeline.mk:238]: Yes continue supporting local config for runs in the config repo, we will need a seaprate solution for that. when we get there. For now leave in

**Review by cpcundill — APPROVED** [review]

**Pull request merged**